liquidity/autoloop_test.go

```diff
@@ -207,7 +207,7 @@ func TestAutoLoopEnabled(t *testing.T) {
 				Events: []*loopdb.LoopEvent{
 					{
 						SwapStateData: loopdb.SwapStateData{
-							State: loopdb.StateInitiated,
+							State: loopdb.StateSuccess,
```

Comment:
Why change the expected change? Did existing autoloop semantics also change?

Reply:
No, first time I introduced `stickyLoopOut` we used to check the `OutRequests` that the mock received, but didn't care about the outcome of the swap itself.

Now I changed the `autoloop` step function to also check that `activeStickyLoops` is `0` before returning. A sticky loop is fetching the swap info from db with the `GetSIngleLoopOut` call which return the value defined in the line above.

`StateInitiated` doesn't really provide any meaningful feedback on the state of the swap so sticky loop out wouldn't return, so I changed it to be `StateSuccess` so that sticky loop outs would immediately return, thus making the `autoloop` step function return with success.

Reply:
Yes I see now, we check `numActiveStickyLoops()` in the test context. Thanks for the explanation!